velociraptor/autoplotter/objects.py

```diff
@@ -1160,7 +1161,7 @@ def create_plots(
                     _, _, exc_traceback = sys.exc_info()
                     print("Traceback:")
                     traceback.print_tb(exc_traceback, limit=10, file=sys.stdout)
-            except UnitConversionError as e:
+            except (UnitConversionError, ConversionError) as e:
```

Comment:
Is there any particular reason why a `ConversionError` would be thrown by the new data?